### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by IlyaGrynyshyn**
image
DockerHub - https://hub.docker.com/r/ilyagrn/wheather_api

### CODE REVIEW

File: .env_sample [hunk 1]
```diff
@@ -0,0 +1 @@
+API_KEY=<YOUR_API_KEY>
```
Comment (viktoria-rybenchuk) [inline .env_sample]: add a blank line in the end of the file

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+LABEL maintainer="ilya@gmail.com"
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py"]
```
Comment (viktoria-rybenchuk) [inline Dockerfile]: the same here

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    city = "Paris"
+    text = requests.get(
+        f"https://api.weatherapi.com/v1/current.json"
+        f"?key={os.getenv('API_KEY')}&q={city}&aqi=yes"
+    ).json()
```
Comment (viktoria-rybenchuk) [inline app/main.py]: write parameters separately as key-value  and then pass them to the request

File: app/main.py [hunk 4]
```diff
@@ -1,6 +1,21 @@
...
+    print(
+        f"Now the weather in {city} is "
+        f"{text['current']['condition']['text']}"
+    )
```
Comment (viktoria-rybenchuk) [inline app/main.py]: did you print the same text as it is in task?

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**